themes/grass/layouts/about/list.html

```diff
@@ -38,6 +38,18 @@ <h2 class="section-title">About GRASS GIS</h2>
             </div>
           </div>
         </div>
+	      <div class="container">
+          <div class="row">
+            <div class="col-12 bg-white mb-4">
+              The Geographic Resources Analysis Support System (<a href="https://grass.osgeo.org/" target="_blank">https://grass.osgeo.org/</a>),
+              commonly referred to as GRASS GIS, is an Open Source Geographic Information System providing powerful raster, vector and geospatial
+              processing capabilities. It can be used either as a stand-alone application or as backend for other software packages such as 
+              <a href="https://qgis.org" target="_blank">QGIS</a> and <a href="https://www.r-project.org/" target="_blank">R</a>
+              geostatistics or in the cloud. It is distributed freely under the terms of the GNU General Public License (GPL).
```

Comment:
```suggestion
              or in the cloud. It is distributed freely under the terms of the GNU General Public License (GPL).
```

Reply:
I suggest to remove geostatistics, since nowadays R is much more than that and many other analysis and visualizations can be done there with geospatial data